scout/parse/variant/variant.py

```diff
@@ -293,5 +293,8 @@ def parse_variant(variant, case, variant_type='clinical',
     mvl_tag = variant.INFO.get('MSK_MVL')
     if mvl_tag:
         parsed_variant['mvl_tag'] = True
+    # use RankScore for somatic variations also
+    somatic_score = parse_rank_score(variant.INFO.get('RankScore', ''), genmod_key)
```

Comment:
Rank score is already parsed and added on line 134 and 135 so this could be removed right?

Reply:
Indeed, removed, thanks.